server/scripts/main_script.py

```diff
@@ -0,0 +1,6 @@
+exec(open("py_add_users.py").read())
+exec(open("add_hunts.py").read())
```

Comment:
also please remember to get the hunt id after you run add_hunts so that you can add the teams to the hunt with that particular id